web_frontend/cloudscheduler/csv2/templates/csv2/clouds.html

```diff
@@ -439,18 +439,6 @@
                                     {% endif %}
                                     
 
-                                    <tr class="highlight3"><td><label for="cacertificate">CA certificate</label></td>
-                                    {% if cloud.cacertificate == None or cloud.cacertificate == "None" %}
-                                        <td><input type="text" name="cacertificate" value="" /></td></tr>
-                                    {% else %}
-                                        <td><input type="text" name="cacertificate" value="{{cloud.cacertificate}}" /></td></tr>
-                                    {% endif %}
-
-                                    <tr class="highlight3"><td><label for="user_domain_name">User domain name</label></td>
```

Comment:
user domain name and project domain name are deleted here. Is there a reason for that?